examples/hosts/app-integration/external-data/README.md

```diff
@@ -38,40 +37,30 @@ Next we need a customer service that functions as the intermediary between the E
+1.POST `/register-session-url`. Required body parameters: `containerUrl` (string), `externalTaskListId` (string). Creates an entry in the Customer Service of the mapping between the container and the external resource id. It then calls the External Service's `/register-for-webhook` endpoint to call it's own `/external-data-webhook?externalTaskListId=XXX` endpoint (details below), in order to notify the Fluid Service containers subscribed to that externalTaskListId of the corresponding change.
 
-Find the details of the API in the [Customer Service README](./src/mock-customer-service/README.md).
-
-Next we come to how the Fluid collaboration session and clients will consume the APIs above and render the data to the screen. This example repo explores two routes to accomplish this. One is the Echo Webhook Pattern and the other is the Bot Pattern. Both are documented in more detail below.
+2. POST `/external-data-webhook`. Required querystring parameters: `externalTaskListId`(string). Registered to be called by the External Data Service webhook when there's been a change to data upstream. On being called, the Customer Service calls the `/broadcast-signal` endpoint in the Fluid Service. Note that this is a route established by the Customer Service to be used exclusively as a subscription endpoint for the external data service's webhook, and should be considered a private implementation detail to the service. Customers may choose to implement this differently.
 
-### Echo Webhook Pattern
+Find the details of the API in the [Customer Service README](./src/mock-customer-service/README.md).
 
-<img width="1356" alt="Scenario: Collaboration session gets an update from outside of Fluid" src="https://user-images.githubusercontent.com/6777404/216415477-14d0b193-29c9-48e5-8b6b-0a549a5dde58.png">
+**Fluid Service**
 
-<img width="1404" alt="Scenario: Collaboration session gets an update from outside of Fluid" src="https://user-images.githubusercontent.com/6777404/216417448-2a43db3e-12a2-48a6-b6d0-a6c4a95e27d1.png">
+_Driver Layer_
 
-<img width="1374" alt="Scenario: EXternal data is updated from within Fluid,if only authenticated users are able to write" src="https://user-images.githubusercontent.com/6777404/216417779-12861504-7909-489c-b7a2-4d75814a396f.png">
+The driver layer section is internal facing and anyone implementing this pattern need not be concerned with this. The changes in the driver layer are a new (and still under construction) `broadcast-signal` endpoint and the logic to relay the signal to the correct Fluid container when called.
```

Comment:
This was a bit hard to parse. Take these suggestions with a grain of salt, but they might make it a bit easier to read.
```suggestion
The driver layer section is internal facing, and anyone implementing this pattern need not be concerned with it. This layer is new (and still under construction). It exposes a `broadcast-signal` endpoint and the logic to relay the signal to the correct Fluid container when called.
```

Reply:
Hmm you know, thinking about it -- it's not true that the implementor doesn't have to be concerned with it since they do need to explicitly call it within their implementation for the customer service. Maybe I should just leave this whole caveatout, except the bit where it's still a bit under construction and might change.